android/settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'sentry_react_native'
+include ':react-native-linear-gradient'
+project(':react-native-linear-gradient').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-linear-gradient/android')
```

Comment:
is this a change that gradle man on you? if so that's fine.

just curious if it's something we asked by our documentation or somewhere else